tools/gen-template-doc/template.adoc.in

```diff
@@ -187,59 +187,22 @@ This template requires the following secrets to be installed for the application
 [[clustering]]
 ==== Clustering
 
-Clustering in OpenShift EAP is achieved through one of two discovery mechanisms:
-Kubernetes or DNS. This is done by configuring the JGroups protocol stack in
-standalone-openshift.xml with either the `<openshift.KUBE_PING/>` or `<openshift.DNS_PING/>`
-elements. The templates are configured to use `DNS_PING`, however `KUBE_PING`is
-the default used by the image.
+Clustering in OpenShift EAP is achieved through discovery mechanisms:
+Kubernetes. This is done by configuring the JGroups protocol stack in
+standalone-openshift.xml with either the `<kubernetes.KUBE_PING/>` element.
+The templates are configured to use `KUBE_PING`.
```

Comment:
@desmax74 could you please update doc more. Right now there is used only one discovery mechanism and as only open shift dns ping was removed. It doesn't sounds well. It would be good to drop the pluralism (e.g. either, mechanism: )

Reply:
ok